src/converters/transactionsConverter.ts

```diff
@@ -141,4 +149,79 @@ export class TransactionsConverter {
             dataItems: dataItems,
         });
     }
+
+    /**
+     * Generally speaking, useful for Relayed V3 transactions.
+     */
+    public transactionOnNetworkToOutcomesOfInnerTransactions(
+        transactionOnNetwork: ITransactionOnNetwork,
```

Comment:
Since the input is a `ITransactionOnNetwork`, this works both for the `TransactionOnNetwork` from the satellite package `sdk-network-providers`, and for the class within `sdk-core/networkProviders` (long time support rationale).